Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/PrefabIntegrationManager.cpp

```diff
@@ -362,14 +379,34 @@ namespace AzToolsFramework
                     {
                         if (!s_prefabFocusPublicInterface->IsOwningPrefabBeingFocused(selectedEntity))
                         {
-                            // Edit Prefab
-                            QAction* editAction = menu->addAction(QObject::tr("Open/Edit Prefab"));
-                            editAction->setShortcut(QKeySequence(Qt::Key_Plus));
-                            editAction->setToolTip(QObject::tr("Edit the prefab in focus mode."));
-
-                            QObject::connect(editAction, &QAction::triggered, editAction, [selectedEntity] {
-                                ContextMenu_EditPrefab(selectedEntity);
-                            });
+                            if (s_prefabPublicInterface->IsOwnedByProceduralPrefabInstance(selectedEntity))
+                            {
+                                // Inspect Prefab
+                                QAction* editAction = menu->addAction(QObject::tr("Inspect Procedural Prefab"));
```

Comment:
I'm ok with this for now, but keep an eye out here, we might want to DRY the action creation / shortcut assignment and just update the labels.

Reply:
Agreed. In general, I'd want to simplify the whole process and have a better system to determine which menu items appear on a specific entity. Current code is very messy and editing it is quite error-prone.